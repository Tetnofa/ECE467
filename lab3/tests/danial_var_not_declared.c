//int z;
int main() { 
    // int x;
    // int z;
    int f;
    //g;
    //int y;
    {
        int z;
        {
            {
                {
                    f;
                    z;
                }
            }
        }
    }
}
void fcn () {
    int y;
    int z;
    int z;
    g;
}
int f;
void poo () {
    f;
}
int z;
int y;
int f;
bool fa () {
    int y;
    int z;
    int g;
    {
        int a;
        {
            a;
        }
    }
}