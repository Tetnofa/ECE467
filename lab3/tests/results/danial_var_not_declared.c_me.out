sema: 23:4 : redefinition of z
sema: 24:4 : use of undefined identifier g
sema: 32:0 : redefinition of f
